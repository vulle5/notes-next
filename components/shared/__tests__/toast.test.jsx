import React from 'react'
import { render } from '@testing-library/react'

import Toast from '../toast'

test('should render button with text', () => {
  const { container } = render(<Toast />)
  expect(container.firstChild).toMatchSnapshot()
})
