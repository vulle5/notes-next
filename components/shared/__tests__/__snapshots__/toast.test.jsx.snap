// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render button with text 1`] = `
<div
  class="jsx-1434399058 container"
>
  <h4
    class="jsx-1434399058"
  >
    This is the message of the toast component
  </h4>
</div>
`;
